res/WELCOME.md

```diff
@@ -21,6 +21,10 @@ Once on the list of permitted usernames, you can contribute the patches to the G
 
 After you submit, GitGitGadget will respond with another comment that contains the link to the cover letter mail in the Git mailing list archive. Please make sure to monitor the discussion in that thread and to address comments and suggestions.
 
+If you want to see what email(s) would be sent for a submit request, add
+a PR comment '/preview' to have the email(s) sent to you.  You must have
+a public GitHub email address for this.
+
```

Comment:
In the `.md` file, please do not wrap the lines of a paragraph. I don't know for certain, but I imagine that GitHub's rendering would retain the wrapped lines, which I imagine would look less elegant.

Reply:
Done and corrected how /preview is displayed.